app/src/main/java/com/example/instagram/PostsAdapter.java

```diff
@@ -61,14 +63,24 @@ public void clear() {
     class ViewHolder extends RecyclerView.ViewHolder {
         private TextView tvUsername;
         private ImageView ivImage;
-        final View rootView;
+        private ImageView ivLikeButton;
+        View rootView;
```

Comment:
Good work on explicitly labeling your instance variable scopes, but why is ```rootView``` here not?

Reply:
Thanks for noticing, I was debugging and forgot to remove line 67.